pkg/pod/status.go

```diff
@@ -109,11 +110,15 @@ func SidecarsReady(podStatus corev1.PodStatus) bool {
 }
 
 // MakeTaskRunStatus returns a TaskRunStatus based on the Pod's status.
-func MakeTaskRunStatus(ctx context.Context, logger *zap.SugaredLogger, tr v1beta1.TaskRun, pod *corev1.Pod, kubeclient kubernetes.Interface, ts *v1beta1.TaskSpec) (v1beta1.TaskRunStatus, error) {
+func MakeTaskRunStatus(ctx context.Context, logger *zap.SugaredLogger, tr v1beta1.TaskRun, pod *corev1.Pod, kubeclient kubernetes.Interface, ts *v1beta1.TaskSpec, spireEnabled bool, spireAPI spire.ControllerAPIClient) (v1beta1.TaskRunStatus, error) {
 	trs := &tr.Status
 	if trs.GetCondition(apis.ConditionSucceeded) == nil || trs.GetCondition(apis.ConditionSucceeded).Status == corev1.ConditionUnknown {
 		// If the taskRunStatus doesn't exist yet, it's because we just started running
 		markStatusRunning(trs, v1beta1.TaskRunReasonRunning.String(), "Not all Steps in the Task have finished executing")
+
+		if spireEnabled {
+			markStatusSignedResultsRunning(trs)
+		}
```

Comment:
why are we marking it as running without checking that the Condition.Status == corev1.ConditionUnknown?

Reply:
or is it assumed that as long as a TaskRun is running, then the signing of results is also running? if so, please add a code comment to clarify

Reply:
We are checking this above. This is the same as markStatusRunning.

Reply:
above, we're checking that apis.ConditionSucceeded.Status == corev1.ConditionUnknown -- what I'm wondering is how this is connected to setting v1beta1.TaskRunConditionResultsVerified.Status to corev1.ConditionUnknown